src/olympia/lib/settings_base.py

```diff
@@ -872,21 +872,10 @@ def JINJA_CONFIG():
             'js/lib/diff_match_patch_uncompressed.js',
             'js/lib/syntaxhighlighter/shCore.js',
             'js/lib/syntaxhighlighter/shLegacy.js',
-            'js/lib/syntaxhighlighter/shBrushAppleScript.js',
-            'js/lib/syntaxhighlighter/shBrushAS3.js',
-            'js/lib/syntaxhighlighter/shBrushBash.js',
-            'js/lib/syntaxhighlighter/shBrushCpp.js',
-            'js/lib/syntaxhighlighter/shBrushCSharp.js',
             'js/lib/syntaxhighlighter/shBrushCss.js',
-            'js/lib/syntaxhighlighter/shBrushDiff.js',
             'js/lib/syntaxhighlighter/shBrushJava.js',
             'js/lib/syntaxhighlighter/shBrushJScript.js',
-            'js/lib/syntaxhighlighter/shBrushPhp.js',
             'js/lib/syntaxhighlighter/shBrushPlain.js',
-            'js/lib/syntaxhighlighter/shBrushPython.js',
-            'js/lib/syntaxhighlighter/shBrushSass.js',
```

Comment:
Would a sass file not be something someone might include as a source that gets compiled to css?